docs/source/item-validation.rst

```diff
@@ -87,6 +87,10 @@ an example of a schema for the quotes item from the :doc:`tutorial </getting-sta
     ]
   }
 
+JSON Schema validation requires to include the location of your JSON schema file
```

Comment:
Improvements in documentation are always welcome! But considering that this branch is to implement Cerberus validation, this change you made is not related. A PR must not mix different subjects.

If you believe this line related to JSON Schema docs is important, the best way to suggest it to the project is creating a separate PR only with this change.

Reply:
This is a change that crept in as I rushed into committing all the changes. 
I was trying to solve the JSONschema issue at the time when I thought it's just a docs issue. 

Removed.